project.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
 
 javadoc.options.links("http://docs.oracle.com/javase/6/docs/api/");
 javadoc.options.links("http://www.javadoc.io/doc/com.google.code.findbugs/jsr305/3.0.1/");
+javadoc.options.links("http://fasterxml.github.com/jackson-core/javadoc/2.2.0/");
```

Comment:
So this doesn't appear to be necessary for JDK 11, and doesn't actually fix it for JDK 8 builds, so boo.